.github/workflows/repackage.yml

```diff
@@ -0,0 +1,47 @@
+name: Package SDK for Bower
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 12 * * *'  # daily @ noon
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout JS SDK
+        uses: actions/checkout@v2
+        with:
+          repository: stellar/js-stellar-sdk
+          fetch-depth: 0
+          persist-credentials: false
+          path: sdk
+
+      - name: Checkout Bower
+        uses: actions/checkout@v2
+        with:
+          repository: stellar/bower-js-stellar-sdk
+          path: sdk/bower
+
+      - name: Install Node (14.x)
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14.x'
+
+      - name: Install and build package
+        run: |
+          cd sdk
+          yarn install
+          yarn
+          cp dist/* bower/
+
+      - name: Publish package
+        run: |
+          TAG=`git describe --tags`
+          cd bower/
+          [ "$(git log -1 --pretty=%B)" = "$TAG" ] && exit
```

Comment:
Explanation: If there's no change between the current latest commit message and the SDK's latest tag, bail. 

This should prevent an empty commit from happening every hour and an early commit happening when the SDK's `master` gets new commits but a release isn't ready yet.

Reply:
Quick question - is it possible to run the check earlier in the process to save some work? If possible maybe run it before you install node?

Reply:
That's a good point :thinking: I think 4c2d6f3 should do it.